creator-node/src/routes/tracks.js

```diff
@@ -34,79 +34,53 @@ module.exports = function (app) {
 
       return errorResponseBadRequest(req.fileFilterError)
     }
+
     const routeTimeStart = Date.now()
-    let codeBlockTimeStart = Date.now()
+    let codeBlockTimeStart
+    const cnodeUserUUID = req.session.cnodeUserUUID
 
-    // create and save track file transcoded version and segments to disk
+    // Create track transcode and segments, and save all to disk
     let transcodedFilePath
     let segmentFilePaths
     try {
+      codeBlockTimeStart = Date.now()
+
       const transcode = await Promise.all([
         TranscodingQueue.segment(req.fileDir, req.fileName, { logContext: req.logContext }),
         TranscodingQueue.transcode320(req.fileDir, req.fileName, { logContext: req.logContext })
       ])
       segmentFilePaths = transcode[0].filePaths
       transcodedFilePath = transcode[1].filePath
 
-      req.logger.info(`Time taken in /track_content to re-encode track file: ${Date.now() - codeBlockTimeStart}ms for file ${req.fileName}`)
+      logger.info(`Time taken in /track_content to re-encode track file: ${Date.now() - codeBlockTimeStart}ms for file ${req.fileName}`)
```

Comment:
@SidSethi why change these to logger from req.logger?

Reply:
this is the new pattern, to decouple logging from the request obj

Reply:
i dont think so. https://github.com/AudiusProject/audius-protocol/blob/master/creator-node/src/components/healthCheck/healthCheckController.js#L15 this passes logger into the component function and it pulls logger off req. also we should probably wait to refactor this to the new pattern until we can do all of it.

Reply:
you right, fixed